custom_dc/upload_data.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Upload Data
+nav_order: 3
+parent: Custom Data Commons
+published: true
+---
+
+## Overview
+
+Raw data (TMCF, CSV) and some other configuration files are uploaded to Google
+Cloud Storage (GCS).
+
+## Storage Layout
+
+All custoom Data Commons data are stored under one GCS folder. Below shows a
+typical layout.
+
+```txt
+.
+└── gcs_folder/
+    ├── data/
+    │   ├── source1/
+    │   │   ├── file_group1/
+    │   │   │   ├── bar.tmcf
+    │   │   │   ├── bar1.csv
+    │   │   │   └── bar2.csv
+    │   │   ├── file_group2/
+    │   │   │   ├── foo.tmcf
+    │   │   │   └── foo.csv
+    │   │   ├── schema.mcf
+    │   │   └── provenance.json
+    │   └── source2/
+    │       ├── file_group1/
+    │       │   ├── baz.tmcf
+    │       │   └── baz.csv
+    │       ├── schema.mcf
+    │       └── provenance.json
+    ├── cache
+    ├── control
+    └── provenance.json
+```
+
+Raw data should be uploaded under `/<project>/data/<import>/<dataset>`. Each
```

Comment:
update the path here to match the above?

Reply:
done